test/test_helpers/consume_assessments_job_test_helper.rb

```diff
@@ -45,6 +45,24 @@ def generic_assessment(symptomatic:)
       }.to_json
     end
 
+    def error_sms_assessment(error_code: TwilioSender::TWILIO_ERROR_CODES[:unknown_error], patient: @patient)
```

Comment:
out of curiosity, when do these 2 methods get called? I see instances of them being called in the consume assessments test, but not sure where they're getting used normally?

Reply:
This is a `consume_assessments_job_test_helper` method so i'm not surprised that it's only being called in the `consume_assessments_job_test`